tests/ceph_installer/test_ansible.py

```diff
@@ -65,6 +66,33 @@ def run(ceph_cluster, **kw):
     build = config.get("build", config.get("rhbuild"))
     ceph_cluster.rhcs_version = build
 
+    # create ssl certificate
+    if (
+        config["ansi_config"]["radosgw_frontend_ssl_certificate"]
```

Comment:
This would throw an `KeyError | AttributeError` when `radosgw_frontend_ssl_certificate` is not passed. Switching to `.get()` would allow us to have the key as optional.

Reply:
@psathyan , sure have made the changes